daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/SBuiltinTest.scala

```diff
@@ -366,7 +364,7 @@ class SBuiltinTest extends AnyFreeSpec with Matchers with TableDrivenPropertyChe
       }
     }
 
-    "Numeric binary operations compute proper results" in {
+    "Decimal binary operations compute proper results" in {
```

Comment:
Why this change? Isn’t Decimal dead at the lf level?

Reply:
it is a glitch.